Riot/Modules/Common/Toasts/RoundedToastView.swift

```diff
@@ -18,47 +18,60 @@ import Foundation
 import UIKit
 import DesignKit
 
-class ActivityIndicatorToastView: UIView, Themable {
+class RoundedToastView: UIView, Themable {
     private struct Constants {
         static let padding = UIEdgeInsets(top: 10, left: 12, bottom: 10, right: 12)
+        static let activityIndicatorScale = CGFloat(0.75)
+        static let imageViewSize = CGFloat(15)
         static let shadowOffset = CGSize(width: 0, height: 4)
         static let shadowRadius = CGFloat(12)
         static let shadowOpacity = Float(0.1)
     }
     
+    private lazy var activityIndicator: UIActivityIndicatorView = {
+        let indicator = UIActivityIndicatorView()
+        indicator.transform = .init(scaleX: Constants.activityIndicatorScale, y: Constants.activityIndicatorScale)
+        indicator.startAnimating()
+        return indicator
+    }()
+    
+    private lazy var imagView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFit
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            imageView.widthAnchor.constraint(equalToConstant: Constants.imageViewSize),
+            imageView.heightAnchor.constraint(equalToConstant: Constants.imageViewSize),
```

Comment:
`Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)`

Reply:
I have my eyes on automatic swift formatter as next improvement